docs/getting-started-guides/rkt/index.md

```diff
@@ -191,29 +160,20 @@ If the cluster is using salt, we can edit the [logging.sls](https://github.com/k
 ##### Check rkt pod status
 
 To check the pods' status, we can use rkt command, such as `rkt list`, `rkt status`, `rkt image list`, etc.
-More information about rkt command line can be found [here](https://github.com/coreos/rkt/blob/master/Documentation/commands.md)
+More information about rkt command line can be found [here](https://github.com/coreos/rkt/blob/master/Documentation/commands.md).
 
 ##### Check journal logs
 
-As we use systemd to launch rkt pods(by creating service files which will run `rkt run-prepared`, we can check the pods' log
-using `journalctl`:
+As we use systemd to launch/manage rkt pods, we can check the pods' log using `journalctl`:
 
 - Check the running state of the systemd service:
 
 ```shell
-$ sudo journalctl -u $SERVICE_FILE
-```
-
-where `$SERVICE_FILE` is the name of the service file created for the pod, you can find it in the kubelet logs.
-
-##### Check the log of the container in the pod:
-
-```shell
-$ sudo journalctl -M rkt-$UUID -u $CONTAINER_NAME
+$ sudo journalctl -u ${SERVICE_NAME}
 ```
 
-where `$UUID` is the rkt pod's UUID, which you can find via `rkt list --full`, and `$CONTAINER_NAME` is the container's name.
+where `${SERVICE_NAME}` is the name of the service file created for the pod, typically the format is `k8s_${RKT_UUID}`.
 
 ##### Check Kubernetes events, logs.
 
-Besides above tricks, Kubernetes also provides us handy tools for debugging the pods. More information can be found [here](/docs/user-guide/application-troubleshooting)
+Besides above tricks, Kubernetes also provides us handy tools for debugging the pods. More information can be found [here](/docs/user-guide/application-troubleshooting).
```

Comment:
s/Besides above tricks, Kubernetes also provides us handy tools for debugging the pods./Kubernetes also provides various tools for debugging pods.